### PR TITLE
Add Issue Templates

### PR DESCRIPTION
**Pull request by prbzrg**
Closes https://github.com/SciML/DifferentialEquations.jl/issues/999

- [X] Bug report
- [X] Feature regression
- [X] Feature request
- [X] Question
- [X] New Algorithm

### CODE REVIEW

**Comment by prbzrg** [comment]
@ChrisRackauckas here are the templates, do we need to add more?

File: .github/ISSUE_TEMPLATE/break_report.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+name: Break report
+about: Create a report to tell us new changes broke something
+title: ''
+labels: 'break'
```
Comment (ChrisRackauckas) [inline .github/ISSUE_TEMPLATE/break_report.md:5]:
```suggestion
name: Feature regression
about: Create a report to tell us about a feature regression, i.e. new changes broke something or resulted in decreased performance
title: '[REGRESSION]: '
labels: 'regression'
```

**Comment by ChrisRackauckas** [comment]
We also tend to have a lot of new algorithm requests, like https://github.com/SciML/OrdinaryDiffEq.jl/issues?q=is%3Aissue+is%3Aopen+label%3Anew-algorithm . It would be good to have this in a form that captures:

* What kinds of problems is the new algorithm useful for
* Other implementations to know about
* References

File: .github/ISSUE_TEMPLATE/feature_regression.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+name: Feature regression
+about: Create a report to tell us about a feature regression, i.e. new changes broke something or resulted in decreased performance
+title: '[REGRESSION]: '
```
Comment (ChrisRackauckas) [inline .github/ISSUE_TEMPLATE/feature_regression.md:4]:
```suggestion
title: ''
```

I guess we don't need things in the title if we are labeling it anyways

**Pull request merged**